### ISSUE TITLE
Using SystemJs

### ISSUE THREAD

**Issue by peteruithoven** [bestiejs/benchmark.js#120]
There is something going wrong when using [Benchmark.js](http://benchmarkjs.com/) with SystemJS / JSPM.  

When installing it with jspm from npm and run the following code, it fails on a line that requires `_` (lodash). So there is probably something going wrong with dependencies. 
``` javascript
import Benchmark from 'benchmark';
console.log('Benchmark(): ', Benchmark());
```

**Comment by jdalton** [comment]
Can you create a JS bin of the issue? Benchmark.js works in Node land.

**Comment by peteruithoven** [comment]
I'm not sure how / if I could do that, JSPM does some transformations when installing packages. I've published a example repo: 
https://github.com/peteruithoven/systemjs-benchmark
Let me know if I can clarify the instructions, I've mainly tried to make it as plug and play as possible.

**Comment by peteruithoven** [comment]
``` javascript
import Benchmark from 'benchmark';
var suite = new Benchmark.Suite;
```
Benchmark 1.0.0 does not throw errors when using the above code. Benchmark 2.0.1 throws the following error: 
```
Uncaught (in promise) Uncaught TypeError: benchmark.Suite is not a function
	Evaluating http://localhost:3001/example3.js
	Error loading http://localhost:3001/example3.js
```
Maybe the API changed, and the documentation isn't updated yet?

**Comment by jdalton** [comment]
That error happens if you don't have lodash loaded as a dependency.
Lodash is the only hard dependency of Benchmark.js.

**Comment by peteruithoven** [comment]
I've tried installing Benchmark (2.0.1) with lodash as dependency:
```
jspm install benchmark -o "{deps: ['lodash']}"
```
and also tried:
```
jspm install benchmark -o "{deps: ['lodash'], globals: {'_': 'lodash'}}"
```
I've also tried adding the following meta config:
``` json
  meta: {
    "benchmark": {
      "deps": [
        "lodash"
      ]
    }
  },
```
Lodash is always installed in my jspm_packages folder. 

If I place a log in the [error handler of the req function](https://github.com/bestiejs/benchmark.js/blob/master/benchmark.js#L647) I get the following error: 
```
Error: Module lodash not declared as a dependency.
```
Which probably comes from SystemJS: https://github.com/systemjs/systemjs/blob/master/lib/register.js#L404

Manually adding regular requires in the code seems to enable lodash and platform as dependencies, but that's not exactly a solution. 

There is a change JSPM 0.17 (beta) handles this better, I'll try that next.
